ossec-server/usr/share/doc/ossec-server/changelog.Debian

```diff
@@ -1,3 +1,10 @@
+ossec-server (2.8.2) unstable; urgency=low
+
+  [ SecureDrop Team ]
```

Comment:
I used the built in helper scripts defaults for updating the changelog. In the packaging guides they do go into a lot of detail on the specifics of the changelog format. I didn't read all of that. So I left what the defaults gave me and added the bare min for people to find out more info.